### PR TITLE
Added documentation for operator layers

### PR DESCRIPTION
**Pull request by graham63**
Add documentation for the Operators supported in LBANN.

### CODE REVIEW

File: docs/operators.rst [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+------------------------------------------------
+Constant Subtract
+------------------------------------------------
+
+Subtract each input value from a constant.
+
+.. math::
+
+   x = (c - x)
```
Comment (benson31) [inline docs/operators.rst:87]:
```suggestion
   y = (c - x)
```

File: docs/operators.rst [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+------------------------------------------------
+Scale
+------------------------------------------------
+
+Scale each input value by a constant.
+
+.. math::
+
+   x = c * x
```
Comment (benson31) [inline docs/operators.rst:136]:
```suggestion
   y = c * x
```

File: docs/operators.rst [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+------------------------------------------------
+Subtract Constant
+------------------------------------------------
+
+Subtract a constant from each input value.
+
+.. math::
+
+   x = (x - c)
```
Comment (benson31) [inline docs/operators.rst:172]:
```suggestion
   y = (x - c)
```

**Review by benson31 — CHANGES_REQUESTED** [review]
Overall this is good, just a few suggestions so it doesn't look like
they're in-place operators (they aren't).

**Review by benson31 — COMMENTED** [review]
Also, do we want to maybe put the Python Front-End names for these in
there? (For both the operator and the OperaorLayer shortcut?)

**Review by bvanessen — APPROVED** [review]

**Pull request merged**